src/Lock.ts

```diff
@@ -54,15 +54,15 @@ export abstract class Lock {
     return this._acquired
   }
 
-  private _startRefresh() {
+  public startRefresh() {
     this._refreshInterval = setInterval(
       this._processRefresh,
       this._refreshTimeInterval
     )
     this._refreshInterval.unref()
   }
 
-  private _stopRefresh() {
+  public stopRefresh() {
```

Comment:
there is no `public` modifier in other public methods

Reply:
fixed